packages/core/scripts/local/DeployEMP.js

```diff
@@ -28,11 +28,12 @@ const argv = require("minimist")(process.argv.slice(), {
   boolean: ["test"],
   string: ["identifier", "collateral", "cversion"]
 });
-const abiVersion = argv.cversion || "1.2.2"; // Default to most recent mainnet deployment, 1.2.2.
+const abiVersion = argv.cversion || "latest"; // Default to most recent mainnet deployment, latest.
```

Comment:
I don't think we should change the default on this script yet since other partners might be using this script on the older factory